guides/gatherers.md

```diff
@@ -327,14 +344,20 @@ values:
 expectations:
   - name: compare_package_corosync
     expect: facts.compare_package_corosync == values.greater_than_installed
+  
+  - name: compare_package_corosync
+    expect: facts.package_corosync.first().version == "2.4.5"
+
+  - name: sbd_version_same_on_all_hosts
+    expect_same: facts.package_sbd.firts().version
```

Comment:
Sorry, me again :smiley: 
typo - firts instead of first

Reply:
super :rocket: